src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java

```diff
@@ -567,6 +572,14 @@ private boolean handleAddedTrip(final Graph graph, final TripUpdate tripUpdate,
             } else {
                 route.setId(new FeedScopedId(feedId, tripId));
             }
+
+            TripDescriptor trip = tripUpdate.getTrip();
+            if(trip.hasExtension(RealtimeExtension.tripDescriptor)) {
+                String url = trip.getExtension(RealtimeExtension.tripDescriptor).getTripUrl();
+                LOG.info("Trip '{}' has URL '{}'", trip.getTripId(), url);
+                route.setUrl(url);
+            }
+
             route.setAgency(dummyAgency);
             // Guess the route type as it doesn't exist yet in the specifications
             route.setType(3); // Bus. Used for short- and long-distance bus routes.
```

Comment:
The route type for carpools is currently 1700 (may change one day...)

Reply:
Yes, I absolutely want to pass it along as a config param.